locator/src/backup_routes.rs

```diff
@@ -15,6 +16,12 @@ pub enum BackupError {
 
     #[error("decode error: {0}")]
     Decode(#[from] bincode::error::DecodeError),
+
+    #[error("gcs error: {0}")]
+    Gcs(#[from] google_cloud_storage::Error),
+
+    #[error("gcs client initialization error: {0}")]
+    GcsInit(String),
```

Comment:
Should this be `#[from] google_cloud_storage::ClientBuilderError`

Reply:
the google_cloud_storage lib doesn't seem to re-export the builder error, and i didn't want to bother adding the other google-cloud-gax dep just for it